upgrade/upgrade-scripts/execute

```diff
@@ -459,6 +478,42 @@ else
 			"for '$ROOTFS_CONTAINER'"
 fi
 
+# In AWS environments, changing instance type could result in generation of
+# different network interface names compared to the existing ones. This mismatch
+# causes our netplan file handler in the osadmin service to remove the existing
+# entries in the netplan plan, because they are not to be found on the new instance.
+# However, the mac address remains consistent and does not change with a change in
+# the instance types.
+# To handle this, from 6.0.16.0 onwards, we are switching to use mac address based
+# interface names in the netplan file. To facilitate this change across upgrades from
+# older verrsions to 6.0.16.0 and above, we create a map from the network interface
+# names to their corresponding mac address, resulting in mac addrress based interface
+# names. For wider context, see DLPX-75209.
+#
+# For deferred upgrades, we cannot know when the engine will be rebooted into the new
+# image. The decision depends on the end-user. Hence we always create the map file
+# irrespective of the upgrade type, to allow the upgrade finalize logic to copy the
+# map file from the currently mounted root filesystem to the root filesystem of the
+# target container.
+#
+# The map creation logic does not apply to engines being upgraded from 6.0.16.0 onwards.
+# This is because the netplan file from 6.0.16.0 will by default be populated with mac
+# address based interface names. This makes this logic redundant for upgrades from
+# 6.0.16.0 and onwards. Hence, we must remove this logic as soon as we mark 6.0.16.0 as
+# the minimum version supported release.
+
+# The container does not have visibility into the complete network namespace of the
+# engine. Hence we need to create the map outside the container to ensure the presense
+# of all the interfaces in the map
+if ! systemd-detect-virt --container --quiet && [[ $(get-appliance-platform) == "aws" ]]; then
+	# While booting back to the original image post rollback, we want to reuse the original
+	# netplan file that consisted of the canonical interface names generated by the kernel.
+	# Hence, we skip creating the map file in rollback execution path
+	if [[ "$1" != "rollback" ]]; then
```

Comment:
IIRC, I don't think this parameter will ever be "rollback".. perhaps you're thinking of the `upgrade` script.. but the `execute` script is only run on upgrade moving forward.. and it's not aware of a rollback event..

so, I think this conditional can be removed.. but please correct me if I'm wrong..

Reply:
I believe your point is correct. Some time back I was trying to figure out the correct script from which to generate this map, and the upgrade script was one of them.
I will remove this check and test both upgrade and rollback once.

Reply:
The check for "rollback" type has been removed.